sys-kernel/installkernel-systemd-boot/files/00-00machineid-directory.install

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+COMMAND="${1}"
+ENTRY_DIR_ABS="${3}"
+
+if ! [[ $KERNEL_INSTALL_MACHINE_ID ]]; then
+    exit 0
+fi
+
+if [[ $COMMAND != add ]]; then
+    exit 0
+fi
+
+# If the machine-id dir does not exist (e.g. $ESP/<machine-id>)
+# create it. It recieves values directly from kernel-install.
+# This is the only function of this plugin.
+MACHINE_ID_DIR="${ENTRY_DIR_ABS%/*}"
+if ! [[ -d "${MACHINE_ID_DIR}" ]]; then
+	if [[ "${KERNEL_INSTALL_VERBOSE}" -gt 0 ]]; then
+    		echo "+mkdir -v -p ${MACHINE_ID_DIR}"
+    		exec mkdir -v -p "${MACHINE_ID_DIR}"
+	else
+    		exec mkdir -p "${MACHINE_ID_DIR}"
```

Comment:
why the exec?

Reply:
same as in other plugins, no other reason